### DISCUSSION TITLE
ios config - exporting to multiple targets

### DISCUSSION THREAD

**Discussion by robertmusialek** [RedMadRobot/figma-export#143] [Q&A]
Hey,

First of all, great tool! However I've run into a small issue:
Is there a way to export the same color assets to multiple targets at once?
Let's say I have 4 targets in my xcode project and 2 figma (let's call them `figmaA` and `figmaB`) files. I want to export colors from `figmaA` so it's added to 2 targets in my xcode proj and from `figmaB` to the remaining 2 targets.

In the .yaml configuration file I found only
```
  # Xcode Target containing resources and corresponding swift code
  target: "UniversalTarget"
```
but it doesn't seem like it supports multiple targets for single export 🤔 

Is this achievable? If yes, how? 

thanks in advance

**Comment by subdan** [comment]
Hi.

Could you explain why do you need to export color styles to the two targets?

You can create a separate target (Dynamic Framework) and export color styles to this target. Then, connect this target to the your targets.

**Comment by robertmusialek** [comment]
Hmm, that is solution I haven't considered.
Essentially we have two brands of one app, each has its own color style. Additionally there are two targets per brand (preprod and prod) which means that each color assets file has to be used in two targets.

I was just wondering if that's doable out of the box. It's mild inconvenience and not really high priority since I can just manually mark check exported files to other targets :)

I'll also explore the dynamic framework solution. Thanks

**Comment by subdan** [comment]
What is the difference between `preprod` and `prod` target? Why you are not using just one target for one brand?

> I was just wondering if that's doable out of the box.

No.

**Comment by robertmusialek** [comment]
> What is the difference between preprod and prod target? Why you are not using just one target for one brand?

mostly configuration files, different info.plist files and they're also on different developer accounts (normal one and enterprise) so having one target per brand is not really viable

**Comment by subdan** [comment]
I recommend to use one target per brand. You can use diffrenent `*.xcconfig` files, Xcode Schemes and Build Configurations for `preprod` and `prod` configurations.

**Reply by robertmusialek** [reply]
will look into that! 
much appreciated
